src/webserver/HttpBase.h

```diff
@@ -161,6 +162,11 @@ class HttpBase
         , lambda_(*this)
         , buffer_(std::move(buffer))
     {
+        BOOST_LOG_TRIVIAL(info) << "[" << tag() << "] http session created";
```

Comment:
Info is probably a bit high for this. In production, these messages will flood the logs, as we get thousands of requests per second.